### ISSUE TITLE
Charms bar not working

### ISSUE THREAD

**Issue by HavocTrooper1** [Icepenguins101/charms-bar-port#10]
I have downloaded all redistributables but somehow the charms bar doesn't work, I have tested the charms bar on both windows 11 22h2 and windows 10 21h2 and it doesn'y show up or work. I have tried Windows key + C and it also didn't work

**Comment by Icepenguins101** [comment]
Is there an error or is it a blank bar?

**Comment by HavocTrooper1** [comment]
Its a blank bar, the charms bar exe is running but it doesnt show up

**Comment by Icepenguins101** [comment]
It’s initializing itself if it ends up looking like this.

**Comment by HavocTrooper1** [comment]
So like how long do I have to wait until the initialization is completed?

**Comment by Icepenguins101** [comment]
When you mouse out and then back for a few seconds it will show up.

**Comment by HavocTrooper1** [comment]
Nope the charms bar doesn’t appear

**Comment by Icepenguins101** [comment]
Are you using a touch device? Touch implementation is partial and will be added on the next release.

**Comment by HavocTrooper1** [comment]
No, I am using a laptop without touch, when I move my mouse to the right and point it down the charms doesn’t appear

**Comment by Icepenguins101** [comment]
I’ll have to examine this when I get to a new update.
